reflect.go

```diff
@@ -270,7 +270,7 @@ func (r *Reflector) reflectTypeToSchema(definitions Definitions, t reflect.Type)
 			rt := &Type{
 				Type: "object",
 				PatternProperties: map[string]*Type{
-					"^[-+]?[0-9]*\\.?[0-9]+$": r.reflectTypeToSchema(definitions, t.Elem()),
```

Comment:
I think it would be better to remove it for now, because unless we can be 100% certain that it allows all JS float formats, this could break downstream packages.